Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG PARENT_VERSION=1.2.19-node16.15.1
+ARG PARENT_VERSION=2.1.0-node16.16.0
```

Comment:
Sommit we need to create tickets to do in all repos

Reply:
We should be regularly updating them as we work on them.  Just like updating packages to deal with vulns frequently, we should do the same with base images

Reply:
Challenge is, how do we do it regularly for inactive repos?